terraform/cp4ba/main.tf

```diff
@@ -9,68 +9,77 @@ data "ibm_resource_group" "group" {
   name = var.resource_group
 }
 
-resource "null_resource" "mkdir_kubeconfig_dir" {
-  triggers = { always_run = timestamp() }
-  provisioner "local-exec" {
-    command = "mkdir -p ${var.cluster_config_path}"
-  }
-}
-
 module "create_cluster" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
-
+//  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
+  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
+//  source = "https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/joel_cp4ba_related_to_issue_276/modules/roks"
   enable               = local.enable_cluster
-  on_vpc               = var.on_vpc
-  project_name         = var.project_name
+  on_vpc               = false
+  project_name         = var.roks_project
   environment          = var.environment
   owner                = var.owner
   resource_group       = var.resource_group
   roks_version         = var.platform_version
+  entitlement          = "cloud_pak"
   flavors              = var.flavors
   workers_count        = var.workers_count
   datacenter           = var.data_center
   force_delete_storage = true
   private_vlan_number  = var.private_vlan_number
   public_vlan_number   = var.public_vlan_number
+  vpc_zone_names       = ["us-south-1"]
```

Comment:
This isn't resolved. `vpc_zone_names` shouldn't be here.

Reply:
Fixed